R/FilterStates.R

```diff
@@ -698,7 +695,7 @@ FilterStates <- R6::R6Class( # nolint
     # @param data_reactive (`function`)
     #  function having `sid` as argument
     #
-    # @return invisible NULL
+    # @return `NULL`, invisibly
```

Comment:
I'm stopping with these. I can do this manually on a commit for all others if they make sense

```suggestion
    # @return `NULL`, invisibly.
```